### PR TITLE
Table Construction Overhaul

### PR DESCRIPTION
**Pull request by uraniummeltdown**
Removed table parts, added table frames.
Wood frame + wood = wood table, + carpet = poker table
Metal frame + metal = normal table, + plasteel = reinf table, + carpet = fancy table, + black carpet = black fancy table, + glass = glass table
Wrench a table frame to decon it
Screwdriver to reduce table to table frame, wrench to decon it straight away
Increased the health (obj_integrity) of some tables and racks
Bonus unrelated change: lowercased the name of carpet

Added /atom/proc/ratvar_act() and ratvar_act() for a few things
Added brass, clockwork floor, brass tables and brass table frames
clock

:cl:
fix: Wooden holodeck table sprites show up properly
tweak: Table construction has been changed a bit, apply stack items (carpet, metal, glass, wood) to table frames to build tables instead of using table parts.
tweak: Slightly increased the health of racks and tables with low health
tweak: Holodeck tables will now smooth with one another
tweak: Holodeck tables and racks can be interacted with in more ways
/:cl:

### CODE REVIEW

**Comment by SamHPurp** [comment]
I keep meaning to do it, but can you make it so that the swedish gene lets you ignore build timer and let them build straight away 👀

**Comment by uraniummeltdown** [comment]
I don't really mind adding that but in case it holds up this PR because of balance memes I'd rather do that in another PR

**Comment by taukausanake** [comment]
> wood table, + carpet = poker table

Isn't that supposed to be grass/astroturf? I don't want to see a maroon carpet turn into a green shag on the table

**Comment by uraniummeltdown** [comment]
in real life poker tables are covered with baize a felt-like fabric not astroturf/grass

**Comment by Jountax** [comment]
Currently poker tables are made by adding astroturf to wooden tables.

**Comment by FantasticFwoosh** [comment]
Obviously you mean importing flat-pack furniture from cargo @SamHPurp , before going down to the canteen to eat some meatballs with cranberries slathered in gravy.

**Comment by FalseIncarnate** [comment]
Those are ligonberries you uncouth heathen!

Jokes aside, this is a good change, makes the system far more standardized and hopefully makes creating new table types easier.

Don’t abductors have a special metal table too? Is that affected?

**Comment by uraniummeltdown** [comment]
@FalseIncarnate Yup I've touched the abductor table too, uses an alien frame + alien alloy to build the table

**Comment by Vivalas** [comment]
If you don't incorporate the swedish meme gene in this I must downvote on principle.

**Comment by davidchan13** [comment]
R tables from plasteel? Bleh, that means Borgs and drones can't make or move them and adds tons of easy to get plasteel too

**Comment by uraniummeltdown** [comment]
It's only 1 plasteel per table

**Comment by davidchan13** [comment]
@uraniummeltdown and with 8 tables around the kitchen thats 4 R walls the clown or some other shitter can drop down basically for free.

**Comment by uraniummeltdown** [comment]
@davidchan13 There are plenty of ways to be a "shitter" in this game, that guy who builds Rwalls to mess with people would just decon the surgery tables or even Rwalls in other parts of the station to do it.
People will notice if the tables around the kitchen are gone, it's Security's job to deal with vandalism/theft in-game and if it's hardcore grief an admin. Rwalls are not invincible, use thermite or just examine the Rwall it tells you how to decon it.

File: code/game/atoms.dm [hunk 1]
```diff
@@ -610,6 +610,9 @@ var/list/blood_splatter_icons = list()
 /atom/proc/narsie_act()
 	return
 
+/atom/proc/ratvar_act()
```
Comment (SamHPurp) [inline code/game/atoms.dm]: But why. This should be self-contained in a Clockwork Cult port, than appended onto a table construction overhaul....

Comment (uraniummeltdown) [reply code/game/atoms.dm]: It'll be too much of a pain to port over something big like ClockCult all at once, Tiger gave me the 👍 to sneaking a bit in

Comment (Aurorablade) [reply code/game/atoms.dm]: its fine to sneak it in as it can cover some bases who ever port its may miss

**Comment by tigercat2000** [comment]
Looks good to me, let's hope this doesn't break anything.

**Pull request merged**